docs/source/advanced/query-batching.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Query batching
+---
+
+Apollo Android supports batching queries as a means to reduce the number of HTTP round trips. When enabled, operations
+executed consecutively in a short period of time will be sent to the server together in a single HTTP request.
+
+Note that this network optimization comes at the price of increased latency as every request in the batch is now as slow
+as the slowest one.
+
+Here is how to enable batching:
+
+```kotlin{5}
+val apolloClient = ApolloClient.Builder()
+    .networkTransport(
+        HttpNetworkTransport.Builder()
+            .serverUrl("https://...")
+            .httpEngine(BatchingHttpEngine())
```

Comment:
There is `ApolloClient.Builder.httpEngine()` accessible

Reply:
Oh even better! 👍